pkg/volume/csi/csi_client.go

```diff
@@ -637,6 +634,14 @@ func (c *csiDriverClient) NodeGetVolumeStats(ctx context.Context, volID string,
 		}
 	}
 
+	usages := resp.GetUsage()
+	// check usage if volume is not abnormal
```

Comment:
I think an error needs to returned only when `!isSupportNodeVolumeCondition && usages == nil`.

Reply:
yes, that case can be considered as well, updated the code for the same.